k9mail-library/src/test/java/com/fsck/k9/mail/store/imap/ImapFolderTest.java

```diff
@@ -42,24 +54,25 @@
 import static org.mockito.Matchers.anyString;
 import static org.mockito.Mockito.doThrow;
 import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.never;
 import static org.mockito.Mockito.times;
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.verifyNoMoreInteractions;
 import static org.mockito.Mockito.when;
 import static org.mockito.internal.util.collections.Sets.newSet;
 
-
-//TODO: Increase test coverage e.g. for fetch() and fetchPart()
 @RunWith(RobolectricTestRunner.class)
 @Config(manifest = Config.NONE, sdk = 21)
 public class ImapFolderTest {
     private ImapStore imapStore;
     private ImapConnection imapConnection;
     private StoreConfig storeConfig;
-
+    private Context context;
 
     @Before
     public void setUp() throws Exception {
+        context = RuntimeEnvironment.application;
+        BinaryTempFileBody.setTempDirectory(context.getCacheDir());
```

Comment:
could just be `BinaryTempFileBody.setTempDirectory(RuntimeEnvironment.application.getCacheDir());`